### PR TITLE
docs(release-info/howto.md): improve typo

### PR DESCRIPTION
**Pull request by Ahhj93**

### CODE REVIEW

File: release-info/howto.md [hunk 1]
```diff
@@ -406,9 +406,9 @@ See the email announce templates in the `templates/` directory.
 
 See
 
-- templates/beta.md for alpha and beta releases
-- templates/rc.md for release candidate
-- templates/production.md for the production release
+- [templates/beta.md](templates/beta.md) for alpha and beta releases
+- [templates/rc.md](templates/rc.md) for release candidate
+- [templates/production.md](templates/production.md) for the production release
```
Comment (dra27) [inline release-info/howto.md:411]: I think this may work with just `[templates/beta.md]`, but I haven't checked.

**Review by dra27 — COMMENTED** [review]
All seems reasonable to me, although I'd possible `s/zsh/sh/` for the syntax, unless there was a subtle aspect of it I'm missing!

Happy, @Octachron?

**Comment by Octachron** [comment]
The formatting change are fine with me, I agree that `s/zsh/sh/g`  would probably be better .

File: release-info/howto.md (hunk 1 above)
Comment (Ahhj93) [reply release-info/howto.md:411]: It doesn't work

**Comment by Ahhj93** [comment]
> The formatting change are fine with me, I agree that `s/zsh/sh/g` would probably be better .

What do I need to change ?

**Comment by dra27** [comment]
All the ```` ```zsh```` to ```` ```sh````, please

**Comment by Ahhj93** [comment]
> All the `` ```zsh `` to `` ```sh ``, please

done

**Review by Octachron — APPROVED** [review]

**Comment by Octachron** [comment]
Thanks for the formatting improvements to the release how-to !

**Pull request merged**